examples/Cargo.toml

```diff
@@ -28,6 +28,19 @@ readme = "README.md"
 [features]
 shared-memory = ["zenoh/shared-memory"]
 unstable = ["zenoh/unstable"]
+transport_shm = ["zenoh/transport_shm"]
+default = [
+    "shared-memory",
+]
+
+# Unfortunately, the feature "transport_shm" is always
```

Comment:
Here there is some commented directives. Are they relevant? Do they hide a misbehaviour/bug that needs to be further investigated?

Reply:
This seems to be a Cargo bug ;)
Here I tried to automatically enable `transport_shm` feature in examples only for Unix and disable for other platforms, but when uncommenting these lines I get zenoh built with `transport_shm` for windows too...

I left this commented block to highlight that there is a good mechanism for making platform-specific features that unfortunately doesn't work, and I plan to create a minimum example and submit an issue for Cargo some day.